### ISSUE TITLE
Update go2rtc on HACS

### ISSUE THREAD

**Issue by sqrg** [AlexxIT/WebRTC#505]
How can I update go2rtc when using WebRTC Camera on Hass?

I can see that it downloaded go2rtc v1.2.0, but I would like to use the latest version

**Comment by AlexxIT** [comment]
You can either use an external go2rtc. Or modify the sources of the integration:
https://github.com/AlexxIT/WebRTC/blob/c79876acb970f3f5027eded1ae0410ef659767b5/custom_components/webrtc/utils.py#L27

**Comment by sqrg** [comment]
Can I make a PR updating that line?

**Comment by AlexxIT** [comment]
No. Some other code also should be updated to use some new go2rtc functions.

**Comment by sqrg** [comment]
Ok, makes sense. Do you think updating go2rtc could help with #500 ?

**Comment by AlexxIT** [comment]
Don't know. PCMA/PCMU awful audio format

**Comment by sqrg** [comment]
But it used to work with WebRTC 2.x, so something must have changed

**Comment by AlexxIT** [comment]
Version 2 counted the timestamp by itself. And did not believe what the camera was sending.
If the camera was sending the wrong timestamp, it might have had some effect.
But for some browsers even a bad timestamp is not a problem.
